app/models/theme.rb

```diff
@@ -327,6 +327,8 @@ def self.expire_site_setting_cache!
       .each do |theme_id|
         Discourse.cache.delete(SiteSettingExtension.theme_site_settings_cache_key(theme_id))
       end
+
+    Discourse.cache.delete(SiteSettingExtension.theme_site_settings_cache_key(nil))
```

Comment:
Must have sucked to debug this.

![](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExZHJya2tpMWxoMjJxZ3Q3M2xxY3MzOXZmYmRmN2tldHI0eWxkYnUzcyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/WxDZ77xhPXf3i/giphy.gif)